src/components/groups/group-create.jsx

```diff
@@ -17,8 +17,8 @@ function GroupCreate() {
   const [formStateObject, setFormStateObject] = useState({
     title: "",
     description: "",
-    createdBy: "todo",
-    modifiedBy: "todo",
+    createdBy: "@TODO:",
```

Comment:
Will it cause an error in the API to not pass a valid date?

Reply:
At one point it did, I am not sure if it does now.